### PR TITLE
Calculate the vsQueueLen based on the Cluster state.

### PR DESCRIPTION
**Pull request by trinaths**
We are updating the VSQueueLen with list of services within the namespaces we are watching. This fixes the issue #810

### CODE REVIEW

File: pkg/appmanager/appManager.go [hunk 1]
```diff
@@ -828,6 +828,15 @@ func (appMgr *Manager) virtualServerWorker() {
 	}
 }
 
+func (appMgr *Manager) getServicesQueueLen() int {
```
[the remark this answers is not in the document]
Comment (trinaths) [reply pkg/appmanager/appManager.go]: Done

File: pkg/appmanager/appManager.go [hunk 2]
```diff
@@ -828,6 +828,15 @@ func (appMgr *Manager) virtualServerWorker() {
...
+	qLen := 0
+	for _, ns := range appMgr.GetWatchedNamespaces() {
+		services, _ := appMgr.kubeClient.CoreV1().Services(ns).List(metav1.ListOptions{})
```
[the remark this answers is not in the document]
Comment (trinaths) [reply pkg/appmanager/appManager.go]: done

**Comment by ssurenr** [comment]
@trinaths Also, please add a statement in the release notes.

File: docs/RELEASE-NOTES.rst [hunk 3]
```diff
@@ -7,6 +7,7 @@ next release
 Bug Fixes
 `````````
 * Controller handles http redirects without entering into an infinite loop.
+* Controller deletes services and recreates them during controller pod restart.
```
Comment (ssurenr) [inline docs/RELEASE-NOTES.rst]:
```suggestion
* :issues:`810` Controller does not delete resources in BIG-IP and recreates during controller pod restart.
```

**Pull request merged**